R/regression.R

```diff
@@ -358,3 +358,76 @@ rae <- function(actual, predicted) {
 explained_variation <- function(actual, predicted) {
     1 - rse(actual, predicted)
 }
+
+#' Mallows's Cp
+#' 
+#' \code{mallowsCp} computes the Mallows's Cp statistic for a linear regression
+#' model. Mallows's Cp is used to assess the fit of an OLS model, often in order
+#' to find the best subsets of the model's predictors. A smaller value of Cp
+#' is preferred, although Cp ~= p is generally accepted. Mallows's Cp is computed
+#' as follows:
+#' 
+#' \deqn{\frac{SSE_\text{sub}}{MSE_\text{full}} - n + 2p_\text{sub}}
+#' 
+#' or, an alternative definition is occasionally given as:
+#' 
+#' \deqn{MSE_\text{sub} + 2p_\text{sub}MSE_\text{full}}
+#' 
+#' While the two definitions don't give the same answer, a model with the lowest
+#' Cp will have the lowest Cp using both definitions.
+#'
+#' @param full_model Either a \code{formula} or \code{lm} for the OLS model using all predictors.
+#' @param sub_model Either a \code{formula} or \code{lm} for the OLS model with a subset of predictors
```

Comment:
Did you consider any variable names other than `sub`? I'm afraid that it won't be obvious to users that `sub` means `subset of columns`? What about `candidate` or `subset`?